drivers/hmis/app/graphql/mutations/set_ho_h_for_enrollment.rb

```diff
@@ -0,0 +1,40 @@
+module Mutations
+  class SetHoHForEnrollment < BaseMutation
+    argument :household_id, ID, required: true
+    argument :client_id, ID, required: true
+
+    field :enrollment, Types::HmisSchema::Enrollment, null: true
+    field :errors, [Types::HmisSchema::ValidationError], null: false
+
+    def resolve(household_id:, client_id:)
+      errors = []
+      enrollment = nil
+
+      client = Hmis::Hud::Client.find_by(id: client_id)
+
+      if client
+        household_enrollments = Hmis::Hud::Enrollment.viewable_by(current_user).where(household_id: household_id)
+        hoh_enrollment = household_enrollments.find_by(relationship_to_ho_h: 1)
+        new_hoh_enrollment = household_enrollments.find_by(personal_id: client&.personal_id)
+        if new_hoh_enrollment
+          hoh_enrollment.relationship_to_ho_h = 99
```

Comment:
I'd be really tempted to make this
```suggestion
          household_enrollments.where( relationship_to_ho_h: 1).update_all(relationship_to_ho_h: 99)
```
That way we can be sure we're enforcing/fixing more than one HoH.
You won't need `hoh_enrollment = ...` on line 17 either.

Reply:
@eanders Great suggestion, this is done